plugins/index.js

```diff
@@ -0,0 +1,23 @@
+import { fileURLToPath } from 'node:url';
+import { dirname } from 'node:path';
+import {BasePlugin} from 'appium/plugin.js';
```

Comment:
Would this import work? Maybe `@appium/base-plugin` needs to be added to dev dependencies?

Reply:
Ah nevermind, this is still a draft. I was confused because CI had run.